src/main/java/frc/trigon/robot/subsystems/pitcher/PitcherConstants.java

```diff
@@ -0,0 +1,160 @@
+    private static final String
+            MASTER_MOTOR_NAME = "MasterPitcherMotor",
+            FOLLOWER_MOTOR_NAME = "FollowerPitcherMotor",
+            ENCODER_NAME = "PitcherEncoder";
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(MASTER_MOTOR_ID, MASTER_MOTOR_NAME, RobotConstants.CANIVORE_NAME),
+            FOLLOWER_MOTOR = new TalonFXMotor(FOLLOWER_MOTOR_ID, FOLLOWER_MOTOR_NAME, RobotConstants.CANIVORE_NAME);
+    static final CANcoderEncoder ENCODER = new CANcoderEncoder(ENCODER_ID, ENCODER_NAME, RobotConstants.CANIVORE_NAME);
+
+    private static final InvertedValue
+            MASTER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            FOLLOWER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final boolean FOLLOWER_MOTOR_OPPOSITE_DIRECTION = false;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final double
+            P = RobotHardwareStats.isSimulation() ? 1 : 1,
+            I = RobotHardwareStats.isSimulation() ? 0 : 0,
+            D = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KS = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KV = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KA = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KG = RobotHardwareStats.isSimulation() ? 0 : 0;
+    private static final double
+            MOTION_MAGIC_ACCELERATION = 0,
+            MOTION_MAGIC_CRUISE_VELOCITY = 0;
+    private static final FeedbackSensorSourceValue ENCODER_TYPE = FeedbackSensorSourceValue.FusedCANcoder;
+    private static final double GEAR_RATIO = 1;
+    private static final SensorDirectionValue ENCODER_SENSOR_DIRECTION_VALUE = SensorDirectionValue.Clockwise_Positive;
+    private static final double ENCODER_MAGNET_OFFSET_VALUE = 0;
+    private static final AbsoluteSensorRangeValue ENCODER_ABSOLUTE_SENSOR_RANGE_VALUE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    static final boolean FOC_ENABLED = true;
+
+    private static final int MOTOR_AMOUNT = 2;
+    private static final DCMotor GEARBOX = DCMotor.getFalcon500Foc(MOTOR_AMOUNT);
+    private static final double
+            PITCHER_LENGTH_METERS = 1,
+            PITCHER_MASS_KILOGRAMS = 0.5;
+    private static final Rotation2d
+            PITCHER_MINIMUM_ANGLE = Rotation2d.fromDegrees(0),
+            PITCHER_MAXIMUM_ANGLE = Rotation2d.fromDegrees(90);
+    private static final SingleJointedArmSimulation SIMULATION = new SingleJointedArmSimulation(
+            GEARBOX,
+            GEAR_RATIO,
+            PITCHER_LENGTH_METERS,
+            PITCHER_MASS_KILOGRAMS,
+            PITCHER_MINIMUM_ANGLE,
+            PITCHER_MAXIMUM_ANGLE,
+            true
+    );
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(0.25).per(Units.Second.of(1)),
+            Units.Volts.of(2),
+            Units.Second.of(1000)
```

Comment:
again, maybe instead of having such a high number, just change the units to minutes?